### DISCUSSION TITLE
Weekly Governance Updates (25 August, 2023)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#811] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

-  FIP0052 - Increase max sector commitment to 3.5 years [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0052.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/475)
- FIP0059 - Synthetic PoRep [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0059.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/649). 
- FIP0067 - PoRep Security Policy & Replacement Sealing Enforcement [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/727)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

* Draft of Removing Restriction to Access to Multiplier and Allowing Unrestricted Minting of Datacap [(PR)](https://github.com/filecoin-project/FIPs/pull/803)
* Extension of Runtime Methods for Address Verification [(PR)](https://github.com/filecoin-project/FIPs/pull/756/commits/3b27c29853e10d300314f30e32ae834e9a404341)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/401)
* Improved Event Syscall API [(PR)](https://github.com/filecoin-project/FIPs/pull/784)
* Draft of Phasing Out Fil+ and Restoring Deal Quality Multiplier to 1x [(PR)](https://github.com/filecoin-project/FIPs/pull/788)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/774)
* Add Cost Opportunity For New Miner Creation [(PR)](https://github.com/filecoin-project/FIPs/pull/796)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/780) 
* First Draft of Market Manual Settlement [(PR)](https://github.com/filecoin-project/FIPs/pull/805)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/800)
* First Draft of Direct Data Onboarding [(PR)](https://github.com/filecoin-project/FIPs/pull/804)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/730)

**Last Call** - *Final opportunity for community feedback!*

FIP0065: Ignore built-in market locked balance in circulating supply calculation [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md) - _Last Call ends EOD 7 September 2023_

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas model adjustment for user programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0063 - Switching to New Drand Mainnet Network [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0063.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/567)
- FIP0070 - Allow SPs to move partitions between deadlines [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0070.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/735)
- FIP0071 - Deterministic Satate Access (IPLD Reachability)(https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0071.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/764) 

 **:fire: HOT :fire: Discussion Topics**

- @kaitlin-beegle posted a public announcement regarding the FIP0001v2 Initiative, a project championed by the Filecoin Foundation on proposed FIP process changes. Read more [here](https://github.com/filecoin-project/FIPs/discussions/799)
- If you are planning to modify sector info fields or looking for opportunities to reduce filecoin state, check out out @ZenGround0's discussion post on State of Sector Info [here](https://github.com/filecoin-project/FIPs/discussions/795). 

---

**nv21 Watermelon network upgrade - FIPs considered in-scope for this upgrade** (_This is a preliminary list and not conclusive_) 
- [FIP0052: Increase Max Sector Commitment to 3.5 years](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0052.md)
- [Add Batch Balancer Parameter Recalibration to FIP0057](https://github.com/filecoin-project/FIPs/pull/673)
- [FIP0059: Synthetic PoRep](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0059.md) 
- [FIP0063: Switching to new DRand mainnet network](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0063.md)
- [FIP0065: Ignore built-in market locked balance in circulating supply calculation](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md)
- [FIP0070: Allow SPs to move partitions between deadlines](https://github.com/filecoin-project/FIPs/discussions/735)
- [SnarkPack SnapDeals](https://github.com/filecoin-project/FIPs/discussions/752)

**Other Governance Updates**
* Join us for the Community Governance Call on **28 August 2023**. Register [here](https://fil-org.zoom.us/meeting/register/tZMkf-2qpjIqEt3nkeKh_7f7_F6wPm76zbTw) if you have not already done so. 
* Follow Filecoin Governance on Twitter [here](https://twitter.com/fil_gov) and get real time updates on the go! 
* Join other Filecoin governance enthusiasts in [this Slack channel](https://filecoinproject.slack.com/archives/C0535S9TUUF)- a place to share and exchange ideas on governing Filecoin. 
* Discussions are currently ongoing for the pending nv21 network upgrade (now called the watermelon 🍉  upgrade), please follow the discussion [here](https://github.com/filecoin-project/core-devs/discussions/140#discussioncomment-6052423)

**Comment by anorth** [comment]
FIP-0047 is superseded by  FIP-0067. Please don't include it in the accepted list any more.
